### PR TITLE
Add PyDMTabWidget

### PR DESCRIPTION
**Pull request by mattgibbs**
PyDMTabWidget is just a QTabWidget that has a channel property for each tab.  The channel is used to display an alarm status indicator for the tab.  Looks like this:
tabs

This PR also changes the default alarm colors across all PyDM widgets.  The new colors are a little brighter, and easier to tell apart, especially for colorblind folks.

### CODE REVIEW

**Comment by mattgibbs** [comment]
Whoops, I forgot to add the files for the example.  Please don't merge this yet.

**Comment by mattgibbs** [comment]
OK, ready for review!

**Comment by hhslepicka** [comment]
Kicked travis again for 3.5.

File: pydm/widgets/tab_bar.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+from pydm.PyQt.QtGui import QTabBar, QTabWidget, QIcon, QBrush, QColor, QVBoxLayout, QWidget, QLabel
+from .base import PyDMWidget
+from .channel import PyDMChannel
+from pydm.PyQt.QtCore import pyqtProperty, Q_ENUMS, Qt, QVariant
+from functools import partial
+from ..utilities.iconfont import IconFont
+
+class PyDMTabBar(QTabBar, PyDMWidget):
```
Comment (hhslepicka) [inline pydm/widgets/tab_bar.py]: Docstrings for class?

**Pull request merged**